websites_list/news_websites.csv

```diff
@@ -1988,3 +1988,13 @@
 1986,New Indian Express,newindianexpress.com,Right Leaning
 1987,Dainik Bhaskar,bhaskar.com,Pro-NDA
 1988,Dainik Jagran,jagran.com,Pro-NDA
+1989,moneycontrol,moneycontrol.com,Ecnonomic news. Owned by TV18
+1990,Bleacher Report,bleacherreport.com
+1991,Eenadu,eenadu.net,Telugu News
+1992,Great Andhra,greatandhra.com,Regional news on Andhra Pradesh
+1993,Dinamalar,dinamalar.com,National Tamil Daily
+1994,Web Dunia,hindi.webdunia.com,Indpendent News Website focused on news in Hindi
+1995,One India,oneindia.com,Leaning right. Owned by BG Mahesh
+1996,Manorama Online,manoramaonline.com,Malyalam newspaper. Generally reliable
+1997,Rediff News,rediff.com/news,Founded by Ajit Balakrishnan
+
```

Comment:
2002,Catholic News Agency,catholicnewsagency.com,Pro-Catholic,Conservative Right
2003,Xinhua News,xinhuanet.com,political,Pro-communist party
2004,Environment News Service,ens-newswire.com,liberal,pro_science
2005,Britain First,britainfirst.org,right_extremist
2006,Kaiser Health News,khn.org,least_biased
2007,Korea Central News Agency,kcna.kp,political,Pro-DPRK Government
2008,The Citizen (South Africa),citizen.co.za,right_center
2009,The New Humanitarian,thenewhumanitarian.org,left_center
2010,Egypt Today,egypttoday.com,right,political
2011,South China Morning Post,scmp.com,left_center
2012,Hong Kong Free Press,hongkongfp.com,political,Anti-chinese government
2013,Asian Correspondent,asiancorrespondent,com,left_center